src/main/webapp/app/modeling-assessment-editor/modeling-assessment-editor.component.html

```diff
@@ -82,14 +84,17 @@ <h4 jhiTranslate="modelingAssessmentEditor.highlightingColors.title">Highlightin
         </div>
     </div>
 </div>
-<div class="mt-3" *ngIf="hasComplaint && result?.id">
-    <h2 class="row-container" jhiTranslate="modelingAssessmentEditor.reviewComplaint">
-        Review Complaint
-    </h2>
+<div class="mt-3" *ngIf="complaint && result?.id">
     <jhi-complaints-for-tutor-form
-        [resultId]="result?.id"
-        [isAllowedToRespond]="result?.assessor.id !== userId"
+        *ngIf="complaint.complaintType === ComplaintType.COMPLAINT"
+        [complaint]="complaint"
+        [isAllowedToRespond]="!isAuthorized"
```

Comment:
Why is it here `!isAuthorized` and below in line `isAuthorized`?

Reply:
Maybe this should be named something like `isAssessor`.

Reply:
Good idea. I used it!